Formula/f/foot.rb

```diff
@@ -0,0 +1,62 @@
+class Foot < Formula
+  desc "Fast, lightweight and minimalistic Wayland terminal emulator"
+  homepage "https://codeberg.org/dnkl/foot"
+  url "https://codeberg.org/dnkl/foot/archive/1.18.1.tar.gz"
+  sha256 "59d22187f7ceaaaa570a5299b102e8f4692826e98785f89ad9d8911802ccc000"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "scdoc" => :build
+
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "libxkbcommon"
+  depends_on :linux
+  depends_on "pixman"
+  depends_on "wayland"
+
+  def install
+    ENV["CFLAGS"] = "#{ENV["CFLAGS"]} -O3"
+
+    mkdir_p "bld/release"
+
+    cd "bld/release" do
+      system "meson",
+        "setup",
+        "-Dterminfo-base-name=foot-extra",
+        "--buildtype=release",
+        "--prefix=#{prefix}",
+        "../.."
+      system "ninja"
+      system "ninja", "test"
+      system "ninja", "install"
+    end
```

Comment:
Let's follow the style used by other formulae, please.
```suggestion
    system "meson", "setup", "build", "-Dterminfo-base-name=foot-extra", *std_meson_args
    system "meson", "compile", "-C", "build", "--verbose"
    system "meson", "install", "-C", "build"
```

Reply:
Although this is much better (my apologies, I should have looked for other examples of meson use). 

Unfortunately, `foot` expects a couple of libraries created by the same author to be present, and when they are not, it falls back to cloning them as subprojects and using that. Since we're using `--wrap-mode=nofallback`, that doesn't happen. These libraries are not packaged in homebrew. 

Is it ok for me to remove the use of `--wrap-mode=nofallback` or do I need to package the other libraries?

Reply:
We can use `--wrap-mode=nodownload`, but we can't remove use of `--wrap-mode` entirely. (It's in `std_meson_args` for a reason, after all.) You'll have to either add the required dependencies as separate formulae or as `resource`s in this one.

Reply:
Ok, I added #192886, and #192887, which will need to be merged first, and updated the dependencies for this formula.